lib/elixir/lib/process.ex

```diff
@@ -399,8 +398,8 @@ defmodule Process do
 
   Inlined by the compiler.
   """
-  @spec demonitor(reference) :: true
   @spec demonitor(reference, options :: [:flush | :info]) :: boolean
+  @spec demonitor(reference, []) :: true
```

Comment:
I am not sure about this particular one; it could help dialyzer, but documentation-wise, it's a bit confusing/redundant IMO. Wdyt? \cc @fishcakez who helped me with this spec when I added `demonitor/2`. :)